src/System.Data.SqlClient/Directory.Build.props

```diff
@@ -1,9 +1,9 @@
-﻿<Project>
+<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <!-- Must be kept in sync with pkg\Microsoft.Windows.Compatibility\Microsoft.Windows.Compatibility.pkgproj -->
     <PackageVersion>4.8.0</PackageVersion>
```

Comment:
Should this be bumped to 4.9.0? Since 4.8.0 was the 3.0 version of this package. @ericstj?

Reply:
3.0 was 4.7.0 https://github.com/dotnet/corefx/blob/5f2c2771e077f98c097ecc1b9202bee22ef50aed/src/System.Data.SqlClient/Directory.Build.props#L5, this should be 4.8.0.

Reply:
Ah right, it confused me that it was not bumped in this change.

Reply:
No worries.  This sort of confusion will hopefully go away in 5.0.  Speaking of which: we should decide what to do with SqlClient in 5.0.  We may want to remove that similar to the Microsoft.Bcl packages.  Can we open an issue and tag sql client owners?